### PR TITLE
Added the EPEL repositories in the FTS installation process with multiline

### PR DESCRIPTION
**Pull request by crsndr**
# Summary

---

**Related issue :**

### CODE REVIEW

**Review by gwarf — APPROVED** [review]
LGTM.
The jscpd check can be ignored as the duplication seem to make sense there.

**Comment by gwarf** [comment]
> The jscpd check can be ignored as the duplication seem to make sense there.

And this was fixed by a commit in another PR.

File: content/en/users/data-transfer/clients/_index.md [hunk 1]
```diff
@@ -26,7 +26,11 @@ Users from other distributions should refer to the
 
 ## Installation
 
-The CLI can be installed from the EPEL repositories with the following package:
+The CLI can be installed from the EPEL repositories for
+[RHEL 7](https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm)
+and
+[RHEL 8](https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm)
```
Comment (andrea-manzi) [inline content/en/users/data-transfer/clients/_index.md:32]:
```suggestion
```

File: content/en/users/data-transfer/clients/_index.md [hunk 2]
```diff
@@ -26,7 +26,11 @@ Users from other distributions should refer to the
 
 ## Installation
 
-The CLI can be installed from the EPEL repositories with the following package:
+The CLI can be installed from the EPEL repositories for
+[RHEL 7](https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm)
+and
```
Comment (andrea-manzi) [inline content/en/users/data-transfer/clients/_index.md:31]:
```suggestion
```

File: content/en/users/data-transfer/clients/_index.md (hunk 1 above)
Comment (andrea-manzi) [reply content/en/users/data-transfer/clients/_index.md:32]: as the client is not available yet in EPEL8 i would remote it

**Comment by enolfc** [comment]
@andrea-manzi can you check this one again? We'd like to introduce some big changes to the structure of the docs and it would be good not to have any pending PR

**Comment by gwarf** [comment]
> @andrea-manzi can you check this one again? We'd like to introduce some big changes to the structure of the docs and it would be good not to have any pending PR

Sorry in fact it's on @crsndr to address the comments by Andrea: basically to remove mention of EPEL-8 as it's not supported.

**Review by andrea-manzi — APPROVED** [review]

**Pull request merged**